PlayerCore/components/ContentBufferingTime.swift

```diff
@@ -0,0 +1,27 @@
+//  Copyright 2019, Oath Inc.
+//  Licensed under the terms of the MIT License. See LICENSE.md file in project root for terms.
+
+import Foundation
+
+public enum ContentBufferingTime {
```

Comment:
We have the same component for mp4 ad buffering time. Maybe we can merge them?

Reply:
If this component is agnostic to whether it is an ad or content - than it makes sense to reuse - and just have separation on toplevel state.

Reply:
Agree, will refactor this.